superset-frontend/src/explore/components/DataTableControl.tsx

```diff
@@ -95,11 +96,30 @@ export const useFilteredTableData = (
     );
   }, [data, filterText]);
 
-export const useTableColumns = (data?: Record<string, any>[]) =>
+export const useTableColumns = (
+  data?: Record<string, any>[],
+  moreConfigs?: { [key: string]: Partial<Column> },
```

Comment:
Extends the possibility of overriding column configs. Not used anywhere for now.